bedrock/firefox/views.py

```diff
@@ -656,6 +665,27 @@ def get_template_names(self):
         return template
 
 
+class WhatsNewEnglishView(WhatsnewView):
+    def get_template_names(self):
+        template = super().get_template_names()
+        if switch('firefox-whatsnew-92-vpn-pricing') and template == ['firefox/whatsnew/whatsnew-fx92-en.html']:
+            template = ['firefox/whatsnew/whatsnew-fx92-vpn-en.html']
+
+        return template
+
+
+class WhatsNewFranceView(WhatsnewView):
+    def get_template_names(self):
+        template = super().get_template_names()
+        locale = l10n_utils.get_locale(self.request)
+        if switch('firefox-whatsnew-92-vpn-pricing') and template == ['firefox/whatsnew/whatsnew-fx92-en.html'] and locale.startswith('en-'):
+            template = ['firefox/whatsnew/whatsnew-fx92-vpn-en.html']
```

Comment:
We shouldn't do this unless you plan on making an English page that's shown to people in France with the right pricing details. From the EU team's perspective I think the only expectation here is a page in French.